assets/translations/en-us.yaml

```diff
@@ -1398,6 +1398,7 @@ cluster:
       httpTokens: Use tokens for metadata
       tagTitle: EC2 Tags
   addOns:
+    dependencyBanner: Add-On Configurations can vary between Kubernetes versions. Changing the Kubernetes version will reset the values below.
```

Comment:
Config is cached given the version of the addon, so changing the kube version won't necessarily reset the config values. Reverting to the previous kube version (and possibly addon config) would also bring back their changes. Suggest something like...

```
Add-On Configurations can vary between Kubernetes versions. Changing the Kubernetes version may reset the values below.
```
...or a more accurate but confusing...
```
Add-On Configurations can vary between Kubernetes versions. Changing the Kubernetes version will update the values below if the Add-On's version has changed.
```